### DISCUSSION TITLE
Quick/in game switch profile

### DISCUSSION THREAD

**Discussion by Shadok38** [opentrack/opentrack#1698] [Ideas]
Hello,
Thank you for this very useful application.
It might be interesting to be able to change profile during the game, something like a keyboard shortcut to profile 1, profile 2, etc...
At the moment I can "automate" this by applying a macro with Voicemacro software, but this involves a short visual return to the desktop during the change.
Thanks

**Comment by sthalik** [comment]
Is anybody other than me willing to implement it?

**Comment by sthalik** [comment]
Caveats:

- for sanity, the profile-switching keystrokes must only work when tracking is already active.
- base the switching code on game detector and global shortcuts (from `opentrack/main-window.[ch]pp`.
